packages/api-client/src/http/BackendErrorLabel.ts

```diff
@@ -74,6 +74,8 @@ export enum BackendErrorLabel {
   NO_ADD_TO_MANAGED = 'no-add-to-managed',
   // Payment errors
   EXPIRED_CARD = 'expired_card',
+  // Domain errors
+  DOMAIN_NOT_FOUND = 'custom-backend-not-found',
```

Comment:
Wouldn't it be better to name the enum after the content?

Reply:
I think it's okay this way because you enter a domain as input param. And in the error case the domain was not found.

Reply:
I don't agree and I think we should stay as close as possible to the backend naming.